asreview/webapp/src/Components/SettingsDialog.js

```diff
@@ -209,7 +209,7 @@ const SettingsDialog = ({ onSettings, toggleSettings }) => {
           variant="subtitle2"
           sx={{ mt: 2, textAlign: "center" }}
           component="a"
-          href="https://asreview.readthedocs.io/en/latest/about.html"
+          href="https://asreview.readthedocs.io/en/stable/"
```

Comment:
Don't you want to refer to https://asreview.readthedocs.io/en/stable/lab/about.html

```suggestion
          href="https://asreview.readthedocs.io/en/stable/lab/about.html"
```